config/blacklight.yml

```diff
@@ -1,9 +1,23 @@
 development:
```

Comment:
blacklight config loader doesn't support YAML anchors. we should maybe try to fix that upstream.